webui/react/src/components/UPlot/UPlotChart/tooltipsPlugin.ts

```diff
@@ -1,6 +1,7 @@
 import uPlot, { Plugin } from 'uplot';
```

Comment:
I am assuming this is what we used to call `tooltipsPlugin2.ts`?

Reply:
:+1: